### PR TITLE
Adding a Dockerfile to build a hellbender container

### PR DESCRIPTION
**Pull request by lbergelson**
Creating a basic dockerfile so we can build containers of hellbender.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# http://phusion.github.io/baseimage-docker/
+FROM phusion/baseimage
+
+RUN  add-apt-repository ppa:webupd8team/java && \
+     add-apt-repository -y ppa:webupd8team/java && \
+     echo debconf shared/accepted-oracle-license-v1-1 select true | sudo debconf-set-selections && \
```
Comment (akiezun) [inline Dockerfile]: sudo here is ok?

Comment (lbergelson) [inline Dockerfile]: It should be, this is completely orthogonal to travis.  These commands are run in a docker vm in order to set up the docker image.

**Comment by akiezun** [comment]
:+1: go ahead
